nengo_viz/static/viz_slider.js

```diff
@@ -393,6 +394,10 @@ VIZ.Slider.prototype.set_range = function() {
     }
 };
 
+VIZ.Slider.prototype.update_range = function(min, max) {
+    this.scale.domain([min, max]);
+}
+
 VIZ.Slider.prototype.layout_info = function () {
     var info = VIZ.Component.prototype.layout_info.call(this);
     info.min_value = this.scale.domain()[1];
```

Comment:
Is this the cause of the [backwards fishiness](https://github.com/nengo/nengo_viz/pull/183/files#diff-7a1005f176b4a21d3b051a0e8dca1fd6R409)? Seems like the min value should be in `[0]` and the max in `[1]`.